CONTRIBUTING.md

```diff
@@ -46,8 +46,27 @@ Finally, if you really, really, _really_ love developing PyBaMM, have a look at
 
 ## Installation
 
-To install PyBaMM with all developer options, type:
+To install PyBaMM with all developer options follow the steps below:
 
+
+1. Install tox using the commands below:
+```bash
+pip install tox # (Using this, you will need to additionally install tox in your virtual environment as decribed below)
+# or
+pipx install tox # (Using this, tox can automatically be accessed in your virtaul environment)
```

Comment:
Would it be worth mentioning that pipx must be installed separately?
Maybe put a link to the pipx project?

Reply:
Yes I'll add some sort of information for pipx.